lib/Doctrine/DBAL/Platforms/OraclePlatform.php

```diff
@@ -1182,4 +1182,30 @@ public function getBlobTypeDeclarationSQL(array $field)
     {
         return 'BLOB';
     }
+
+    public function getListTableCommentsSQL(string $table, ?string $database = null) : string
+    {
+        $table = $this->normalizeIdentifier($table);
+        $table = $this->quoteStringLiteral($table->getName());
+
+        $tableCommentsName = 'user_tab_comments';
+        $ownerCondition    = '';
+
+        if ($database !== null && $database !== '/') {
+            $database          = $this->normalizeIdentifier($database);
+            $database          = $this->quoteStringLiteral($database->getName());
+            $tableCommentsName = 'all_tab_comments';
+            $ownerCondition    = ' AND owner = ' . $database;
```

Comment:
Please use `quoteStringLiteral()` immediately during concatenation and leave the variable unchanged. Otherwise, it's not reusable (see the comment above).

Reply:
:+1: done